Functions.Templates/Templates/EdgeHubTrigger-CSharp/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+    "defaultFunctionName": "EdgeHubTrigger",
+    "description": "$EdgeHubTrigger_description",
+    "name": "Azure IoT Edge Hub trigger",
+    "language": "C#",
+    "category": [
+      "$temp_category_core",
+      "$temp_category_dataProcessing"
+    ],
+    "categoryStyle": "iotEdge",
```

Comment:
Change to `iot`

Reply:
Updated, could you please share why we need to use `iot` instead of `iotEdge` here?